src/overlays/actors/ovl_En_Pr2/z_en_pr2.c

```diff
@@ -723,7 +723,8 @@ void func_80A75950(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Acto
     }
 }
 
-s32 func_80A759D8(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx, Gfx** gfx) {
+s32 EnPr2_OverrideLimbDraw2(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx,
```

Comment:
Same here. This would be the generic variant, while the other one would be `Opa` (I would probably add `Opa` to the post limb draw as well).